packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -117,9 +117,18 @@ You can also reference these org-level attributes, such as:
 
 All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
 
-#### Email templates for customization
+<ApiLifecycle access="ie" /><br>
+
+#### Access to app context
+
+Okta Identity Engine orgs can now call app context within emails via Velocity Templating Language. When an end user enters an authentication flow, OIE stores that app context in the state token, which can then be called using the following app macros:
+
+* `$app.id`
+* `$app.name`
+* `$app.label`
+
+When these macros are combined with conditional logic, you can trigger per app branding, and strings to uniquely customize a single email template to dynamically change based on the app your end user visited, which triggered the email. These app macros are not available on Classic Engine due to the state token not existing.
```

Comment:
A few more suggestions:

"When used with conditional logic, you can trigger per app branding and define strings to uniquely customize an email template based on the app your end user was using when the email was triggered. App context is not available on Classic Engine since the state token does not exist there."

Reply:
resolved thanks